lib/timeline/component/Group.js

```diff
@@ -50,12 +61,17 @@ function Group (groupId, data, itemSet) {
 Group.prototype._create = function() {
   var label = document.createElement('div');
   if (this.itemSet.options.groupEditable.order) {
-	  label.className = 'vis-label draggable';
+    label.className = 'vis-label draggable';
   } else {
-	  label.className = 'vis-label';
+    label.className = 'vis-label';
   }
   this.dom.label = label;
 
+  var collapseGroupBtn = document.createElement('div');
```

Comment:
I think the whole `collapseGroupBtn`can be removed, now that the arrow is done is CSS.